src/tools/snapshot.ts

```diff
@@ -198,14 +202,34 @@ const screenshot: Tool = {
   handle: async (context, params) => {
     const validatedParams = screenshotSchema.parse(params);
     const tab = context.currentTab();
-    const options: playwright.PageScreenshotOptions = validatedParams.raw ? { type: 'png', scale: 'css' } : { type: 'jpeg', quality: 50, scale: 'css' };
-    const screenshot = await tab.page.screenshot(options);
+    const fileName = path.join(os.tmpdir(), sanitizeForFilePath(`page-${new Date().toISOString()}`)) + `${validatedParams.raw ? '.png' : '.jpeg'}`;
+    const options: playwright.PageScreenshotOptions = validatedParams.raw ? { type: 'png', scale: 'css', path: fileName } : { type: 'jpeg', quality: 50, scale: 'css', path: fileName };
```

Comment:
go for default quality here (remove param)

Reply:
This also had `options` previously. Without the param we can't specify file format + file location to save the screenshot.